dotcom-rendering/src/web/components/StickyBottomBanner.importable.tsx

```diff
@@ -207,7 +211,12 @@ export const StickyBottomBanner = ({
 	switches,
 	pageId,
 	keywordsId,
-}: Props) => {
+}: Props & {
+	switches: Switches;
+	isSensitive: boolean;
+}) => {
```

Comment:
It feels like at this point that the two sets of props for `StickyBottomBanner` and `StickyBottomBannerWithAB` are similar enough that they should both use the same props type?

Perhaps it should be defined as 
```
type Props = {
    ...
} & ABProps;
```

Then we can change both `Props & { switches: Switches; isSensitive: boolean; }` and `Props & ABProps` to just `Props`

Then just make sure we use the correct naming scheme (`pageIsSensitive`, `abTestSwitches`) in both components.